airflow/utils/logging.py

```diff
@@ -137,6 +151,20 @@ def __init__(self):
                 '"{}". Please make sure that airflow[gcp_api] is installed '
                 'and the GCS connection exists.'.format(remote_conn_id))
 
+    def exists(self, remote_log_location):
```

Comment:
Would be good to name this something a bit more descriptive:
def log_exists(self, log_location)